micro-rdk/src/esp32/exec.rs

```diff
@@ -9,7 +9,7 @@ use futures_lite::{
 pub struct Esp32Executor {}
 
 std::thread_local! {
-    static EX: LocalExecutor<'static> = LocalExecutor::new();
+    static EX: LocalExecutor<'static> = const { LocalExecutor::new() };
```

Comment:
Native clippy with new nightly rust made this suggestion for the native executor, so I made it here too just for consistency.